exercises/ruby-1-a/ruby_1_a_example.rb

```diff
@@ -1,11 +1,17 @@
-VOWELS = %w{a e i o u}
-def word_with_most_vowels(sentence)
-  words = sentence.split(" ")
-  word_vowels = words.map do |word|
-    [
-      word,
-      word.downcase.chars.select{|l|VOWELS.include?(l)}.count
-    ]
+class ShoppingCart
```

Comment:
For the second part of the experiment, there will be three starting points, each using different approaches. Could you perhaps add two new example implementations, using different approaches? That way, you immediately have the starting points for the second part of the exercise, and a verification that it can indeed be solved using different approaches.

Reply:
So the plan is to have three examples for the same test suite: `ruby_1_a_example_1.rb, ruby_1_a_example_2.rb, ruby_1_a_example_3.rb`, right?

Reply:
Correct!

Reply:
Ok, added another two examples keeping the same interface and tests.